packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -602,7 +602,7 @@ contract OPContractsManager is ISemver {
 
     /// @notice addGameType deploys a new dispute game and links it to the DisputeGameFactory. The inputted _gameConfigs
     /// must be added in ascending GameType order.
-    function addGameType(AddGameInput[] memory _gameConfigs) external returns (AddGameOutput[] memory) {
+    function addGameType(AddGameInput[] memory _gameConfigs) public virtual returns (AddGameOutput[] memory) {
```

Comment:
Changing `external` to `public` would result in a bytecode change, and thus the semver bump should be minor.